### PR TITLE
Add CrossCloud

### PR DESCRIPTION
**Pull request by chrihech**
CrossCloud is a sync application for arbitrary cloud storages (Dropbox, Google Drive, etc.) as well as any other data storage source with an API (CRM, Windows shares, etc.). 

We are very happy with what we could build with Electron and would like to share it. 
The GUI is entirely created using Electron/React. Not sure which evidence to include, simply download the .app from https://crosscloud.me/download and navigate to /Contents/Resources/app to see the GUI source :-) 

thanks for this repo - helped us a lot!

### CODE REVIEW

**Comment by sindresorhus** [comment]
Do you have a link to a blog post or something with a screenshot of the app we could link to instead? Most people here will not bother downloading and will just leave if they don't see how it looks like.

File: readme.md [hunk 1]
```diff
@@ -165,6 +165,7 @@ Made with Electron.
 - [Freeter](https://freeter.io) - Organizer for freelancers & creatives.
 - [WhatsApp](https://www.whatsapp.com/download/) - Official WhatsApp app.
 - [CatLight](https://catlight.io) - Build status notifier.
+- [CrossCloud](https://crosscloud.me) - All your cloud storage in one sync application + client side encryption.
```
Comment (sindresorhus) [inline readme.md]: Can you shorten the description?

**Comment by sindresorhus** [comment]
ping :arrow_up:

**Comment by chrihech** [comment]
Hi @sindresorhus, 
Sorry for the late response. Yes, here we have a blog post showing a screenshot of the GUI (https://medium.com/@crosscloud/what-is-new-in-crosscloud-1-0-c00404e7bf9f#.kvdbis1iz). I just updated the link in cross-cloud:master. Would it be possible to reopen the PR? 

thanks

Christoph

**Pull request closed without merging**